log.go

```diff
@@ -78,19 +84,52 @@ func (el *eventLogger) EventBegin(ctx context.Context, event string, metadata ..
 	start := time.Now()
 	el.Event(ctx, fmt.Sprintf("%sBegin", event), metadata...)
 
+	span, ctx := opentrace.StartSpanFromContext(ctx, event)
+
 	eip := &EventInProgress{}
 	eip.doneFunc = func(additional []Loggable) {
-
+		if traceingDisabled() {
```

Comment:
This will disable normal event logs if tracing is disabled. Lets move this check down until after the call to `el.Event`

Reply:
doh! Yes it would.